File: DRRR.Server/Dtos/ChatRoomPasswordValidationResponseDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace DRRR.Server.Dtos
{
    /// <summary>
    /// 验证房间密码响应DTO
    /// </summary>
    public class ChatRoomPasswordValidationResponseDto
    {
        /// <summary>
        /// 错误信息
        /// </summary>
        public string Error { get; set; }

        /// <summary>
        /// 是否需要刷新
        /// </summary>
        public bool? RefreshRequired { get; set; }
    }
}

File: DRRR.Client/src/app/chat-rooms/dtos/chat-room-password-validation-response.dto.ts
/**
 * 验证房间密码请求DTO
 */
export interface ChatRoomPasswordValidationResponseDto {
  /**
   * 验证房间密码响应DTO
   */
  error: string;

  /**
   * 是否需要刷新
   */
  refreshRequired: boolean;
}
